src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -25,6 +25,10 @@ private BoundExpression BindInterpolatedString(InterpolatedStringExpressionSynta
             var stringType = GetSpecialType(SpecialType.System_String, diagnostics, node);
             var objectType = GetSpecialType(SpecialType.System_Object, diagnostics, node);
             var intType = GetSpecialType(SpecialType.System_Int32, diagnostics, node);
+
+            // PROTOTYPE follow up with Neal on the implementation 
```

Comment:
```suggestion
            // PROTOTYPE(CONSTISTR) follow up with Neal on the implementation 
```

Reply:
This has been done, you can remove the comment :)

---
In reply to: [447263586](https://github.com/dotnet/roslyn/pull/45396#discussion_r447263586) [](ancestors = 447263586)

Reply:
Indeed, the comment has been removed.

---
In reply to: [447335348](https://github.com/dotnet/roslyn/pull/45396#discussion_r447335348) [](ancestors = 447335348,447263586)